README.md

```diff
@@ -1,23 +1,57 @@
+# Change Tracking Plugin for SAP Cloud Application Programming Model (CAP)
+
+[![REUSE status](https://api.reuse.software/badge/github.com/cap-js/change-tracking)](https://api.reuse.software/info/github.com/cap-js/change-tracking)
+
+The `@cap-js/change-tracking` package is a [CDS plugin](https://cap.cloud.sap/docs/node.js/cds-plugins#cds-plugin-packages) providing out-of-the box support for automatic capturing, storing, and viewing of the change records of modeled entities as simple as that:
+
+1. [Install the plugin: `npm add @cap-js/change-tracking`](#setup)
+2. [Add `@changelog` annotations to your CDS models](#annotations)
+3. [Et voilà:](#change-history-view)
+
+<img width="1300" alt="change-history-custom" src="_assets/changes-custom.png">
+
 
-The `@cap-js/change-tracking` package is a [CDS plugin](https://cap.cloud.sap/docs/node.js/cds-plugins#cds-plugin-packages) providing out-of-the box support for automatic capturing, storing, and viewing of the change records of modeled entities.
 
 ### Table of Contents
 
+- [Preliminaries](#preliminaries)
 - [Setup](#setup)
-- [Usage](#usage)
-  - [Add `@changelog` Annotations](#add-changelog-annotations)
-    - [Human-readable Types and Fields](#human-readable-types-and-fields)
-    - [Human-readable IDs](#human-readable-ids)
-    - [Human-readable Values](#human-readable-values)
-  - [Test-drive locally](#test-drive-locally)
-  - [Change History View](#change-history-view)
-  - [Customizations](#customizations)
+- [Annotations](#annotations)
+  - [Human-readable Types and Fields](#human-readable-types-and-fields)
+  - [Human-readable IDs](#human-readable-ids)
+  - [Human-readable Values](#human-readable-values)
+- [Test-drive locally](#test-drive-locally)
+- [Change History View](#change-history-view)
 - [Contributing](#contributing)
-  - [Code of Conduct](#code-of-conduct)
+- [Code of Conduct](#code-of-conduct)
 - [Licensing](#licensing)
 
 
 
+## Preliminaries
+
+In this guide, we use the [Incidents Management reference sample app](https://github.com/cap-js/incidents-app) as the base to add change tracking to. Clone the repository and apply the step-by-step instructions:
+
+```sh
+git clone https://github.com/cap-js/incidents-app
+cd incidents-app
+npm i
+```
+
+**Alternatively**, you can clone the incidents app including the prepared enhancements for change-tracking:
+
+```sh
+git clone https://github.com/cap-js/calesi --recursive
```

Comment:
not public yet

Reply:
@danjoa: Let's take this out until made public?

Reply:
or we make it public now ;)

Reply:
cap-js/calesi is public now